rero_ils/modules/items/models.py

```diff
@@ -63,3 +63,12 @@ class ItemCirculationAction(Enum):
     CHECKIN = 'checkin'
     REQUEST = 'request'
     EXTEND = 'extend'
+
+
+class ItemNoteTypes(Enum):
```

Comment:
We had a discussion the other day, it seems the `Enum` does not really add a value for these types. I put it as `object`, anyway, I will have a PR later on to move to object for those where `Enum` does not add a value.

Reply:
Done